outputs/US/costs_2020.csv

```diff
@@ -4079,12 +4079,12 @@ lignite,efficiency,0.33,p.u.,"Lazard's levelized cost of energy analysis - versi
 lignite,investment,3827.1629,EUR/kW_e,"Lazard's levelized cost of energy analysis - version 16.0 (2023): https://www.lazard.com/media/typdgxmm/lazards-lcoeplus-april-2023.pdf , pg. 49 (Levelized Cost of Energy - Key Assumptions), accessed: 2023-12-14.","Higher costs include coal plants with CCS, but since using here for calculating the average nevertheless. Calculated based on average of listed range, i.e. (3200+6775) USD/kW_e/2 / (1.09 USD/EUR). Note: Assume same costs as for hard coal, as cost structure is apparently comparable, see https://diglib.tugraz.at/download.php?id=6093e88b63f93&location=browse and https://iea.blob.core.windows.net/assets/ae17da3d-e8a5-4163-a3ec-2e6fb0b5677d/Projected-Costs-of-Generating-Electricity-2020.pdf .",2023.0,,
 lignite,lifetime,40.0,years,"Lazard's levelized cost of energy analysis - version 16.0 (2023): https://www.lazard.com/media/typdgxmm/lazards-lcoeplus-april-2023.pdf , pg. 49 (Levelized Cost of Energy - Key Assumptions), accessed: 2023-12-14.","Note: Assume same costs as for hard coal, as cost structure is apparently comparable, see https://diglib.tugraz.at/download.php?id=6093e88b63f93&location=browse and https://iea.blob.core.windows.net/assets/ae17da3d-e8a5-4163-a3ec-2e6fb0b5677d/Projected-Costs-of-Generating-Electricity-2020.pdf . ",2023.0,,
-methanation,FOM,3.0,%/year,"Agora Energiewende (2018): The Future Cost of Electricity-Based Synthetic Fuels (https://www.agora-energiewende.de/en/publications/the-future-cost-of-electricity-based-synthetic-fuels-1/), section 6.2.3.1",,2017.0,,
+methanation,FOM,3.0,%/year,Schaber thesis, from old pypsa cost assumptions,2015.0,,
 methanation,carbondioxide-input,0.198,t_CO2/MWh_CH4,"Götz et al. (2016): Renewable Power-to-Gas: A technological and economic review (https://doi.org/10.1016/j.renene.2015.07.066), Fig. 11 .",Additional H2 required for methanation process (2x H2 amount compared to stochiometric conversion).,,,
 methanation,efficiency,0.8,per unit,Palzer and Schaber thesis, from old pypsa cost assumptions,2015.0,,
 methanation,hydrogen-input,1.282,MWh_H2/MWh_CH4,,Based on ideal conversion process of stochiometric composition (1 t CH4 contains 750 kg of carbon).,,,
-methanation,investment,777.5294,EUR/kW_CH4,"Agora Energiewende (2018): The Future Cost of Electricity-Based Synthetic Fuels (https://www.agora-energiewende.de/en/publications/the-future-cost-of-electricity-based-synthetic-fuels-1/), table 6: “Reference scenario”.",,2017.0,,
-methanation,lifetime,20.0,years,Guesstimate.,"Based on lifetime for methanolisation, Fischer-Tropsch plants.",2017.0,,
+methanation,investment,1058.2216,EUR/kWH2,Schaber thesis, from old pypsa cost assumptions,2015.0,,
+methanation,lifetime,25.0,years,Schaber thesis, from old pypsa cost assumptions,2015.0,,
```

Comment:
Intentional or unintentional change by removing methanation from the manual_inputs.csv?

Reply:
see above